docker-deploy/README.md

```diff
@@ -0,0 +1,26 @@
+### How to start with Docker compose
+##### Require : 
+##### + Install docker-compose for your server
+##### + Download java-jre8 at https://www.oracle.com/java/technologies/javase-jre8-downloads.html (Product / File Description: Linux x64 RPM Package) Then rename file download to jre8-latest.rpm and copy to docker-deploy/backend
+```bash
+$ git clone https://github.com/VietOpenCPS/Vaccom.git && cd docker-deploy
+```
+##### Edit variable in docker-compose.yml then run docker-compose
```

Comment:
List all parameters need to be modified here.

Reply:
Thanks, I fixed it

Reply:
I see the public IP address "119.17.200.21" should be one of thing necessary to be changed case by case.